README.md

```diff
@@ -20,6 +20,7 @@ Development builds are also available from [our website](https://pcsx2.net/downl
 | Operating System | CPU | GPU | RAM |
 | --- | --- | --- | --- |
 | - Windows 7 or newer (32 or 64 bit) <br/> - Ubuntu 18.04/Debian or newer, Arch Linux, or other distro (32 or 64 bit) | - Supports SSE2 <br/> - [PassMark Single Thread Performance](https://www.cpubenchmark.net/singleThread.html) rating near or greater than 1600 <br/> - Two physical cores, with hyperthreading | - Direct3D11 support <br/> - OpenGL 3.x support <br/> - [PassMark G3D Mark](https://www.videocardbenchmark.net/high_end_gpus.html) rating around 3000 (GeForce GTX 750) <br/> - 2 GB Video Memory | 4 GB |
+*Note: Recommended Single Thread Performance is based on moderately complexity games. Games that pushed the PS2 hardware to its limits will struggle on CPUs at this level. Some release titles and 2D games which underutilized the PS2 hardware may run on CPUs rated as low as 1200.*
```

Comment:
moderately complexity

Reply:
You mean, change into "moderately complexed"?
Yes, thats seems grammatically correct.

Reply:
Isn't complex already an adjective? You don't need to derive it from the (pretty rare) verb. 
Anyway, how about wording it a bit more "hopefully"? Like "may run on CPUs even rated lower than 1200."

> So which is it?

Turns out I'm a moron and indeed, with my 1300 and something cpu, it's only getting 70% speed *most of times*. 
I.. guess like I had just limited myself to menus in the past on the emulator?

> I gave an explanation of where the number came from. It's just something we felt out based on our experiences.

Yes, and I would have liked for that to be better.. shared I guess? But whatever I guess.